packages/paste-website/src/pages/form/input/index.mdx

```diff
@@ -0,0 +1,578 @@
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Prefix/suffix</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Background: $color-background</ListItem>
+          <ListItem>Border: $color-border-light</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Prefix/suffix text</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Default: $color-text</ListItem>
+          <ListItem>Disabled: $color-text-weak</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Value text</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>$font-size-30</ListItem>
+          <ListItem>Default, Readonly: $color-text</ListItem>
+          <ListItem>Disabled: $color-text-weak</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No, but it can take any children</td>
+    </tr>
+    <tr>
+      <td>Placeholder text </td>
+      <td>
+        $color-text-weak, italic
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Help text</td>
+      <td>
+        $color-text-weak, $font-size-30
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Inline error</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Text: $color-text-error, $font-size-30</ListItem>
+          <ListItem>Icon: IconError, $color-text-error, $icon-size-30</ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <td>Spacing</td>
+      <td>
+        <UnorderedList marginBottom="space0">
+          <ListItem>Label:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Bottom: $space-10</ListItem>
+              <ListItem>Between label and required indicator: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Input:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Left padding: $space-40</ListItem>
+              <ListItem>Top, right, bottom padding: $space-30</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Prefix/suffix padding: $space-30</ListItem>
+          <ListItem>Help text:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Top: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+          <ListItem>Inline error:
+            <UnorderedList marginBottom="space0">
+              <ListItem>Spacing between icon and text: $space-20</ListItem>
+              <ListItem>Top: $space-20</ListItem>
+            </UnorderedList>
+          </ListItem>
+        </UnorderedList>
+      </td>
+      <td>No</td>
+    </tr>
+  </tbody>
+</table>
+
+---
+
+## Usage Guide
+
+### API
+
+#### Installation
+
+```bash
+yarn add @twilio-paste/form
+```
+
+#### Usage
+
+```jsx
+import {FormInput, FormLabel, FormHelperText} from '@twilio-paste/form';
```

Comment:
This is a slight departure from our usual. Normally we show a little code snippet of the composition.

Reply:
Just forgot to add it back in. I'll get it added.